apps/web/components/ActivatePoints.tsx

```diff
@@ -47,6 +72,21 @@ export function ActivatePoints({
     functionName: "deactivatePoints",
   });
 
+  useWaitForTransaction({
+    hash: deactivatePointsData?.hash,
```

Comment:
the same here, works fine?

Reply:
![image](https://github.com/1Hive/gardens-v2/assets/13890618/29e7905c-07d9-4127-a02b-43fe1c0ff4e5)

Reply:
![image](https://github.com/1Hive/gardens-v2/assets/13890618/ca107796-13d0-4902-ae9c-f3abbf0f396b)